packages/arb-util/hashing/hashing.go

```diff
@@ -74,6 +90,10 @@ func Uint32(input uint32) []byte {
 	return solsha3.Uint32(input)
 }
 
+func Uint32Array(input []uint32) []byte {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported function Uint32Array should have comment or be unexported](https://app.codacy.com/gh/OffchainLabs/arbitrum/pullRequest?prid=5428340)